vMenu/menus/VehicleOptions.cs

```diff
@@ -1702,7 +1704,25 @@ async void HandleItemSelect(Menu menu, MenuItem menuItem, int itemIndex)
                 if (vehicleExtras.TryGetValue(item, out var extra))
                 {
                     var veh = GetVehicle();
-                    veh.ToggleExtra(extra, _checked);
+                    if (veh != null && veh.Exists())
```

Comment:
```suggestion
                    if (Entity.Exists(veh))
```

Using `Entity.Exists` does both a check against `null` and makes sure the vehicle actually exists 👍 

Reference: https://github.com/citizenfx/fivem/blob/master/code/client/clrcore/External/Entity.cs#L1419-L1422